CHANGELOG

```diff
@@ -56,6 +56,23 @@ Miscellaneous
   the application level.  Example usage of this new interface can be found
   in the examples/sundials directory.
 
+- Added support for creating refined versions of periodic meshes, making use of
+  the new L2ElementRestriction class. This class also allows for computing
+  geometric factors on periodic meshes using partial assembly.
+
+- New method Mesh::GetHilbertElementOrdering for sorting mesh elements along
+  the Hilbert curve. The ordering can be used to improve caching and parallel
+  partitioning in non-conforming AMR.
+
+- New options to reorder and partition the mesh in the mesh-explorer miniapp.
+
+- Improved element numbering after uniform mesh refinement.
+
+New and updated examples and miniapps
+-------------------------------------
+- The (p)mesh-optimizer miniapp has been updated to demonstrate mesh optimization
+  for an AMR mesh.
+
```

Comment:
Some of the above text comes from different PRs and was merged here incorrectly -- e.g. there are bullets that are repeated twice.

Reply:
This is fixed now